### DISCUSSION TITLE
Error using  int arr[n]; in c++ codes

### DISCUSSION THREAD

**Discussion by infilev** [microsoft/vscode-cpptools#9925] [General]
I am getting error while performing the below action(in the screenshot)

![issue1](https://user-images.githubusercontent.com/107412360/192795938-731c538f-d06b-4e32-aa6a-9a62b7fe8ff5.png)

![issue2](https://user-images.githubusercontent.com/107412360/192795976-5830b7bb-3952-4290-b691-3e998ec857e3.png)

**Comment by sean-mcmanus** [comment]
Why did you post a PDF? Can you just inline paste the screenshot or code sample?

**Reply by infilev** [reply]
I have attached the Screenshot of the issue.

**Comment by infilev** [comment]
I have figured out the problem after some internet surfing and it was either related to 'compiler path'  or 'intelliSenseMode' in 'c_cpp_properties.json'.
Previous my settings were as follows.

![issue2](https://user-images.githubusercontent.com/107412360/192849137-edff3b59-41c5-4a1f-b7b3-cb52058ba3e7.png)


And now the updated settings are

![solution1](https://user-images.githubusercontent.com/107412360/192849030-ecad1949-f372-4884-8aac-fa10a9aa50d4.png)

And more thing to add that I think 'cl.exe' doesn't support it( i.e variable length array).

**Reply by sean-mcmanus** [reply]
Yes, you're correct. cl.exe doesn't support the variable length array.
